src/main/java/javaserver/RequestHandlers/DirectoryHandler.java

```diff
@@ -0,0 +1,14 @@
+package javaserver.RequestHandlers;
```

Comment:
It's strange that the routes get configured in `Routes.java` and in each individual request handler. That means that if a route name changes, there would be two places that would require this change.